block_engine/src/block_engine.rs

```diff
@@ -167,8 +132,6 @@ impl BlockEngineRelayerHandler {
                                 datapoint_error!("block_engine_relayer-error",
                                     "block_engine_url" => block_engine_url,
                                     "auth_service_url" => auth_service_url,
-                                    "cluster" => &cluster,
```

Comment:
why remove?

Reply:
its part of the host_id now, no need to manually pass it around

Reply:
there were a bunch of metrics that didnt have cluster and region, grafana couldnt filter well